llama-index-integrations/vector_stores/llama-index-vector-stores-wordlift/manager_client/api/vector_search_queries_api.py

```diff
@@ -0,0 +1,312 @@
+# coding: utf-8
+
+"""
+    Manager
+
+    Subscription management and related services.
+
+    The version of the OpenAPI document: 1.0
+    Contact: hello@wordlift.io
+    Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+    Do not edit the class manually.
+"""  # noqa: E501
+
+import warnings
+from pydantic import validate_call, Field, StrictFloat, StrictStr, StrictInt
+from typing import Any, Dict, List, Optional, Tuple, Union
+from typing_extensions import Annotated
+
+from manager_client.models.page_vector_search_query_response_item import PageVectorSearchQueryResponseItem
+from manager_client.models.vector_search_query_request import VectorSearchQueryRequest
+
+from manager_client.api_client import ApiClient, RequestSerialized
+from manager_client.api_response import ApiResponse
+from manager_client.rest import RESTResponseType
+
+
+class VectorSearchQueriesApi:
```

Comment:
Any plans to publish this as its own package? Like a client SDK? Its fine if it lives here for now though

Reply:
yes, we’ll publish the API as its own package in the future